app/src/components/pages/collectionLanePage/collectionLanePage.tsx

```diff
@@ -3,29 +3,57 @@ import {
   Box,
   Heading,
   HorizontalRule,
+  Pagination,
 } from "@nypl/design-system-react-components";
-import { useParams } from "next/navigation";
+import {
+  useParams,
+  usePathname,
+  useRouter,
+  useSearchParams,
+} from "next/navigation";
 import { headerBreakpoints } from "../../../utils/breakpoints";
 import {
+  displayResults,
   slugToString,
+  totalNumPages,
   createAdobeAnalyticsPageName,
 } from "../../../utils/utils";
-import { mockCollections } from "__tests__/__mocks__/data/mockCollections";
 import { CardsGrid } from "../../grids/cardsGrid";
 import React, { useEffect, useRef, useState } from "react";
 import PageLayout from "../../pageLayout/pageLayout";
-import useBreakpoints from "@/src/hooks/useBreakpoints";
-import { useTooltipOffset } from "@/src/hooks/useTooltipOffset";
 import LaneLoading from "../../lane/laneLoading";
 
-export default function CollectionLanePage() {
+export default function CollectionLanePage({ data }: any) {
   const params = useParams();
   const slug = params.slug as string;
   const title = slugToString(slug);
   const [isLoaded, setIsLoaded] = useState(false);
-  const { isLargerThanLargeTablet } = useBreakpoints();
-  const cardRef = useRef<HTMLDivElement>(null);
-  const tooltipOffset = useTooltipOffset(cardRef);
+
+  const pathname = usePathname();
+  const queryParams = useSearchParams();
+
+  const [currentPage, setCurrentPage] = useState(
+    Number(queryParams.get("page")) || 1
+  );
+
+  const { push } = useRouter();
+
+  const totalPages = totalNumPages(data.numResults, data.perPage);
+
+  const headingRef = useRef<HTMLHeadingElement>(null);
+
+  const updatePageURL = async (pageNumber: number) => {
+    const params = new URLSearchParams();
+    params.set("page", pageNumber.toString());
+    setCurrentPage(pageNumber);
+    const url = `${pathname}?${params.toString()}`;
+    setIsLoaded(false);
+    push(url);
```

Comment:
What I'm seeing locally is:

- I click on "next"
- the skeleton loaders display
- the focus goes to the text "displaying ..."
- then the page URL updates and the new data comes in

Let's keep investigating this but the order might need to move around or the timeout should be longer.